doc/guides/decoding/decoder_options.rst

```diff
@@ -91,6 +91,19 @@ processing applied after decoding to a NumPy :class:`~numpy.ndarray`:
 Miscellaneous Options
 =====================
 
+The following options may be used with both native (uncompressed) and encapsulated
+(compressed) transfer syntaxes when decoding to a NumPy :class:`~numpy.ndarray`:
+
+* `apply_shift_correction`: :class:`bool` - if ``True`` (default) and *Bits Stored*
```

Comment:
I would prefer something like `clear_unused_bits` that shows what you are doing, not how you are doing it. Your call, though... (yes, I know that "clear" may sound like "fill with zero", which is not true for signed data, but still...)

Reply:
Changed it to `correct_unused_bits`

Reply:
Yeah, even better, thanks!